task-2/src/usage.ts

```diff
@@ -0,0 +1,39 @@
+import { Inventory } from "./Inventory";
+import { Item } from "./Item";
+import { ItemWeightComparator } from "./ItemWeightComparator";
+import { Pizza } from "./Pizza";
+import { Sword } from "./Sword";
+
+// Create the inventory
+const inventory: Inventory = new Inventory();
+
+// Create a set of items
+const a: Item = new Sword(30.4219, 0.7893, 300, 2.032);
```

Comment:
I just left this file as it was. But I don't think that `a` should have `Item` type, it should be `Sword`

Reply:
I think it has an `Item` type to verify if `Sword` can be treated as an `Item`. But actually, it doesn't make sense, because this is already verified in `inventory.addItem(a)`. So yeah, it's better to be a `Sword`